cucoslib/dispatcher/flows/githubManifestMetadataFlow.yml

```diff
@@ -0,0 +1,30 @@
+---
+  flow-definitions:
+    - name: 'githubManifestMetadataFlow'
+      node_args_from_first: true
+      queue: '{DEPLOYMENT_PREFIX}_githubManifestMetadataFlow_v0'
+      edges:
+        - from:
+          to: 'GitHubManifestMetadataInit'
+        - from: 'GitHubManifestMetadataInit'
+          to: 'metadata'
+        - from: 'metadata'
+          to: 'dependency_snapshot'
+        - from: 'dependency_snapshot'
+          to: 'bayesianFlow'
+          foreach:
+            import: 'cucoslib.dispatcher.foreach'
+            function: 'iter_dependencies_analysis'
+            propagate_result: true
+          condition:
```

Comment:
You sure about this? When I removed those it looked like I wasn't able to control whether I want to analyze transitive dependencies or not.